### DISCUSSION TITLE
sectionproperties Video chat meetup

### DISCUSSION THREAD

**Discussion by connorferster** [robbievanleeuwen/section-properties#152] [General]
Hi @robbievanleeuwen, @ccaprani, @Spectre5, @normanrichardson, @Agent6-6-6 , @Czarified,

I am thinking it would be fun to do a video chat meet-up. I mentioned it to Robbie and he is in. Someday between now and January 3rd at a time that works across our many time zones. I believe many of us are either on the North American continent or the Oceania continent so I would propose 5pm (PST)  / 8pm (EST) / 12pm (AEDT).

If folks are interested let me know what day(s) could work for you and I can post a Whereby meeting room link. I am available any of those days except Dec 31.

**Comment by ccaprani** [comment]
Sounds great, 1/2 Jan will work for me. Happy to go late night Aussie time if that helps?

**Comment by robbievanleeuwen** [comment]
Very keen! I'm very free over the next few days until the 3rd Jan Australian time 👍

**Comment by normanrichardson** [comment]
Great, on my end I can do any of those days.

**Comment by Spectre5** [comment]
Plan it without me, but I'll join if I can.  I hope everyone has a great New Year!

**Comment by connorferster** [comment]
Ok! I will set it up for the following time:

```python
import pytz
import datetime

PST = pytz.timezone("Canada/Pacific")
AEDT = pytz.timezone("Australia/Sydney")
CST = pytz.timezone("US/Central")
mtg_time = PST.localize(datetime.datetime(2022, 1, 1, 17, 0, 0, 0))

print("PST: ", mtg_time)
print("AEDT: ", mtg_time.astimezone(AEDT))
print("CST: ", mtg_time.astimezone(CST))

>>>  PST:  2022-01-01 17:00:00-08:00
     AEDT:  2022-01-02 12:00:00+11:00
     CST:  2022-01-01 19:00:00-06:00
```

If you are in another timezone (other than the ones listed), copy and paste the code above and create your own timezone instance for your particular timezone. If you have not used `pytz` before (like me, one hour ago...), you can find a list of all the timezone names by running `pytz.all_timezones` (no parens)

At that time, go to: https://connorferster.whereby.com/sectionproperties
(Nothing to install, runs in your browser on any device with audio/camera)

**Reply by robbievanleeuwen** [reply]
haha love it! see you then!

**Reply by ccaprani** [reply]
That's awesome! `datetime` and timezones are so complex...there's a great listen on it here: https://talkpython.fm/episodes/show/271/unlock-the-mysteries-of-time-pythons-datetime-that-is

**Comment by connorferster** [comment]
Love that episode!

**Comment by Czarified** [comment]
Hey guys! I'm excited for what y'all have been accomplishing here! Sorry I missed out and have been MIA. I was off the grid for the end of the year. Trying to keep up with all the changes overall! Lot's of great work! Happy new year!
